scenarios/web/small/core/2.1/mvc/mvc.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0-preview2-30008" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Sdk" Version="2.1.0-preview2-30008" />
```

Comment:
Why is this PackageReference required?  Will it be required by customers who want to enable `RazorCompileOnBuild` when 2.1 ships?  If so, so it be included in `Microsoft.AspNetCore.App`?

Reply:
This will be part of the web sdk in preview2. That work is currently in progress.